train.py

```diff
@@ -306,17 +309,18 @@
 
                 model.train()
 
-logger.info(f"Done training, best model saved to {best_model_path}")
+logger.info(f"Trained for {num_epochs} epochs, best model at {best_model_path}")
 
-logger.info("Loading best model: %s" % best_model_path)
-best_model = torch.load(best_model_path)
-model.load_state_dict(best_model)
+if best_model_path is not None:
+    logger.info("Loading best model: %s" % best_model_path)
+    best_model = torch.load(best_model_path)
+    model.load_state_dict(best_model)
 
-full_eval = WorldCerealEval(train_df, val_df)
-results = full_eval.finetuning_results(model, model_modes=["Random Forest", "Regression"])
-logger.info(json.dumps(results, indent=2))
-if wandb_enabled:
-    wandb.log(results)
+    full_eval = WorldCerealEval(train_df, val_df)
```

Comment:
Why do we move this into the `if` statement? 

We might still want to run / see the results locally, no?

Reply:
This is just github displaying the changes in a weird way, the code looks like this
```python

if best_model_path is not None:
    logger.info("Loading best model: %s" % best_model_path)
    best_model = torch.load(best_model_path)
    model.load_state_dict(best_model)

    full_eval = WorldCerealEval(train_df, val_df)
    results = full_eval.finetuning_results(model, model_modes=["Random Forest", "Regression"])
    logger.info(json.dumps(results, indent=2))
    if wandb_enabled:
        wandb.log(results)

if wandb_enabled and run:
    run.finish()
    logger.info(f"Wandb url: {run.url}")

```

Reply:
hah confusing. Sorry about that